core/src/main/resources/lib/layout/task.jelly

```diff
@@ -201,7 +206,8 @@ THE SOFTWARE.
                       ${attrs.badge.text}
                     </span>
                   </j:if>
-                </a>
+                </m:a>
+                <st:adjunct includes="lib.layout.task.task" />
```

Comment:
Ditto, why not simply
```suggestion
                <st:adjunct includes="lib.layout.task"/>
```
plus `core/src/main/resources/lib/layout/task/task.js` ⇒ `core/src/main/resources/lib/layout/task.js` rename?

Reply:
As above.